src/sentry/search/events/fields.py

```diff
@@ -954,8 +980,8 @@ def normalize(self, value, params):
 
 
 class ColumnNoLookup(Column):
-    def __init__(self, name, allowed_columns=None):
-        super().__init__(name, allowed_columns=allowed_columns)
+    def __init__(self, name, **kwargs):
+        super().__init__(name, **kwargs)
```

Comment:
Is this some intermediate change that made it into the PR? I don't see it being used anywhere.

Reply:
just leftover changes cause I was originally going to extend `allowed_columns` until I realized that doing so wouldn't work with tags. 

gonna just remove this init entirely now I'm reading it again since `ColumnNoLookup` doesn't do anything here